pkgs/by-name/xe/xeus/package.nix

```diff
@@ -37,11 +37,13 @@ stdenv.mkDerivation (finalAttrs: {
   doCheck = true;
   preCheck = ''export LD_LIBRARY_PATH=$PWD'';
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { };
```

Comment:
You need `nix-update-script` as input on the top.

Reply:
Also, I do see some instance of the following (no use of `with`):
```nix
  meta = {
    maintainers = [ lib.maintainers.eelco ];
    description = "A utility for Nixpkgs contributors to check Nixpkgs for common errors";
    mainProgram = "nixpkgs-lint";
    platforms = lib.platforms.unix;
  };
```

Reply:
I believe both are fine and it's up to personal preference.

Reply:
Sounds good!

Thanks for your help. Feel free to let me know if there's anything else needed for approval.